### DISCUSSION TITLE
Change cellClassName after grid integration

### DISCUSSION THREAD

**Discussion by greg59650** [nick-keller/react-datasheet-grid#202] [Q&A]
Is it possible to change the class of a cell when retrieving an object?

I create my array columns like this:

const columns = [
    { ...keyColumn('active', checkboxColumn), title: 'Active',cellClassName: MyCellComponent },
    { ...keyColumn('firstName', textColumn), title: 'First name' , cellClassName: MyCellComponent },
    { ...keyColumn('lastName', textColumn), title: 'Last name', cellClassName: MyCellComponent },
  ]

MyCellComponent determines the class I add to my cell.

  const MyCellComponent = ({
    rowData,
    rowIndex,
    columnId,
    result,
  }: {
    rowData: any;
    rowIndex: number;
    columnId?: string;
    resultIntegration: any;
  }) => {
     return 'int';
  };

Once my array is installed, I click on a button which returns a "result" object which returns a list of column id and index row.

I mate the "result" object with the rowIndex and columnId values, to change the class of my cell.

The problem is that MyCellComponent doesn't get the result object back after integration.

Is there a solution to change the cellClassName on certain events?

**Comment by nick-keller** [comment]
Hey @greg59650 ,
Can you please provide a code example that I can look at?
